crates/messages/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "mc-probufs-messages"
```

Comment:
up for suggestions on the common message format crate, it could be that `types` is a better name :shrug:

Reply:
renamed back to messages